vendor/github.com/aws/aws-sdk-go-v2/service/cloudwatch/service.go

```diff
@@ -0,0 +1,80 @@
+// Code generated by private/model/cli/gen-api/main.go. DO NOT EDIT.
+
+package cloudwatch
+
+import (
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/signer/v4"
```

Comment:
@kaiyan-sheng I had a look at the imports and compared them to what is checked in here. It seems for example this package here is not checked in? Could it be that you have it locally but not versioned? This could explain why the binary is so big compared to the changed code lines.

Reply:
I don't think the compiler will put anything in the binary if it's not being used. For example, when you remove the module from `include.go` the resulting binary goes back to ~65MB.

Do you know how we can debug this to understand the reason for this extra ~15MB? Perhaps @urso knows this?

@ruflin how concerned are you about this? Perhaps we can go for it's own beat if you think it's a big deal

Reply:
@ruflin hmm for aws/signer/v4? This is already in https://github.com/elastic/beats/blob/master/vendor/vendor.json#L231 and https://github.com/elastic/beats/tree/master/vendor/github.com/aws/aws-sdk-go-v2/aws/signer/v4

Reply:
@kaiyan-sheng Got it, that is why some of the dependency are missing.

@exekias As I assume adding future metricsets to it will not increase the size I'm ok with getting it in. But I would still like to understand why it increases much more then I think it should.